### DISCUSSION TITLE
请教一下通过BLE连接需要怎么配置从机的特征值？

### DISCUSSION THREAD

**Discussion by Loser9527** [wh201906/SerialTest#82] [Q&A] [answered]
![serial](https://github.com/user-attachments/assets/cf5b2aeb-244e-4f65-99bd-591aa38c1059)
如图，蓝牙从机模块可以被搜索并连接，但是选好了服务跟特征值后点打开，从机端就会断连，回到广播状态等待连接。
手机上的其它蓝牙调试软件包括微信小程序都是可以正常通过该特征值读写数据的。
是不是需要特别配置从机的某些设置呢？

**Comment by wh201906** [comment]
不应该啊？你用的是哪款模块？

**Reply by Loser9527** [reply, marked as answer]
我在模块的软件包里找到一个无线串口的例程，能用了，应该是我自己写的软件有什么地方没配置好，app是没问题的！

**Reply by wh201906** [reply]
能用就好
